### ISSUE TITLE
SyntaxError: Unexpected token 'R', "Rate limited." is not valid JSON

### ISSUE THREAD

**Issue by afilbert** [CircleCI-Public/CircleCI-Env-Inspector#13]
I'm trying to fetch payload for our organization, and apparently running into a rate limit.

**Input output:**

```
? Enter your CircleCI API token [hidden]
? Select a VCS GitHub
? Enter your GitHub API token [hidden]
? Select an account REDACTED
? Is this an Organization (Not a User)? Yes
```

**Error output:**

```
undefined:1
Rate limited.
^

SyntaxError: Unexpected token 'R', "Rate limited." is not valid JSON
    at JSON.parse (<anonymous>)
    at Response.json (file:///project/node_modules/node-fetch/src/body.js:149:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async fetchWithToken (file:///project/utils.js:45:24)
    at async file:///project/index.js:122:40
    at async Promise.all (index 137)
    at async file:///project/index.js:119:18

Node.js v19.3.0
```

**Comment by flywheel-danger** [comment]
My bad, this should be closed as dup of #5

**Comment by afilbert** [comment]
Closing as dup of #5
